wrapper/src/main/java/software/amazon/jdbc/plugin/AwsSecretsManagerConnectionPlugin.java

```diff
@@ -120,7 +120,7 @@ public AwsSecretsManagerConnectionPlugin(PluginService pluginService, Properties
     final Region region = Region.of(regionString);
     if (!Region.regions().contains(region)) {
       throw new RuntimeException(Messages.get(
-          "AwsSecretsManagerConnectionPlugin.unsupportedRegion",
+          "IamAuthConnectionPlugin.unsupportedRegion",
```

Comment:
I propose we change the message name to something more generic, maybe AwsSdk.unsupportedRegion or AwsAuth.unsupported...? A bit weird to have an IamAuthConnPlugin message in the secrets manager plugin.